### DISCUSSION TITLE
196 fps Eternal Light

### DISCUSSION THREAD

**Discussion by dannymarsbar** [rocksdanister/lively#1209] [General]
I used interpolation to increase of the eternal light wallpaper: https://drive.google.com/file/d/1Ahq2QdqDKyrPMEXUFuGEDjGCi2i-op9s/view?usp=sharing

**Comment by rocksdanister** [comment]
That's cool, which software did you use?

Its actually possible to do this realtime using mpv (player used for video wallpaper): https://github.com/rocksdanister/lively/issues/1144

**Reply by dannymarsbar** [reply]
I used Flowframes, easy to use and great AI
